src/modm/platform/core/cortex/module.lb

```diff
@@ -164,7 +165,7 @@ def prepare(module, options):
             description=FileReader("option/main_stack_size.md"),
             minimum=2 ** 8,
             maximum=2 ** 16,
-            default="3*1024"))
+            default="1024"))
```

Comment:
Maybe this is the reason for [the following error](https://github.com/modm-io/modm/pull/439/checks?check_run_id=1678887253):
```
...
Linking········ build\release\sk6812.elf

scons: building terminated because of errors.
c:/gcc-arm-none-eabi-10-2020-q4-major/bin/../lib/gcc/arm-none-eabi/10.2.1/../../../../arm-none-eabi/bin/ld.exe:modm/link/linkerscript.ld:205
cannot move location counter backwards (from 20001100 to 20001000)

collect2.exe: error: ld returned 1 exit status
scons: *** [build\release\sk6812.elf] Error 1
```

Reply:
Increasing `main_stack_size` for the sk6812 example does not fix the error.

Reply:
The linker cannot fit all static data into RAM, so something requires too much RAM.

Reply:
`examples/nucleo_f031k6/sk6812` from current develop with 8*4 LEDs:
```
Program:   6.0 KiB (18.9% used)
(.build_id + .data + .fastcode + .fastdata + .hardware_init + .rodata +
 .table.copy.intern + .table.heap + .table.zero.intern + .text + .vector_rom)

Data:      1.8 KiB (45.9% used) = 856 B static (20.9%) + 1024 B stack (25.0%)
(.bss + .data + .fastdata + .stack)

Heap:      2.2 KiB (54.1% available)
(.heap1)
```

With fiber (this branch) and 8*8 LEDS:
```
Program:   6.4 KiB (19.9% used)
(.build_id + .data + .fastcode + .fastdata + .hardware_init + .rodata +
 .table.copy.intern + .table.heap + .table.zero.intern + .text + .vector_rom)

Data:      3.9 KiB (96.7% used) = 2936 B static (71.7%) + 1024 B stack (25.0%)
(.bss + .data + .fastdata + .stack)

Heap:    132.0 B   (3.2% available)
(.heap1)
```

Reply:
This PR makes the main function a fiber with its own [2048 byte stack](https://github.com/modm-io/modm/pull/439/files#diff-e5ffc2ccd70736e5d4c6c60378428599ac385769462198c5bec9561c45c64d48R17). That's why we are getting roughly a 2K ram usage increase here.